util/mqtt_test.go

```diff
@@ -0,0 +1,22 @@
+package util
+
+import (
+	"testing"
+
+	"github.com/planetmint/planetmint-go/x/dao/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSendMqttMessages(t *testing.T) {
+	t.Parallel()
+	t.Skip("Skip this test case as this test case expects a working MQTT connection" +
+		"the test case is intended to work manually.")
```

Comment:
What does it mean: `the test case is intended to work manually.`? I don't understand this.

Reply:
eckelj wrote: we didn't reveal the MQTT credentials yet. that's why I wanted to skip the test in automation for the time being.